Modules/System/SharePoint/src/model/listitem/SharePointListItem.Codeunit.al

```diff
@@ -88,4 +90,9 @@ codeunit 9103 "SharePoint List Item"
             SharePointListItem."List Id" := SharePointUriBuilder.GetMethodParameter('Lists').Substring(6, 36);
         end;
     end;
+
+    [IntegrationEvent(false, false)]
```

Comment:
Same for this event. 
Events of Internal Objects can't subscribed from other apps.